### ISSUE TITLE
Installation fails

### ISSUE THREAD

**Issue by onlinebene** [scaleway-community/scaleway-cozy#20]
It seems cozy has changed their installer and the automatic installation of the cozy cloud is blocked at the point when the type of SSL certificate has to be chosen.

**Comment by solarus0** [comment]
Hello world.
I encountered the same problem.

After the VM’s installation, the `apt-get update `don’t work due to a mispelled file and a duplicate source in the sources.list.d folder.
After that, a` dkpg-reconfigure` set the TLS certificate, but the nginx server desn’t respond anyway on the 443 port.
At least I tried to `apt-get update` and now the nginx server send me a Bad Gateway when I connect.

I will check the sites-avaibles and sites-enabled when I have time, to see where the problem come from.

**Comment by onlinebene** [comment]
Hi, 

To relaunch the blocked cozy cloud installation, you may type the following commands as a temporary bugfix: 

`pkill apt-get
rm -fr /var/lib/dpkg/lock
rm /var/cache/debconf/config.dat
rm /var/cache/debconf/passwords.dat
rm /var/cache/debconf/templates.dat
dpkg --configure `

**Comment by solarus0** [comment]
Hello,
I tried to reconfigure broken packets but it still doesn’t work.

```
Setting up cozy (1.24) ...
Chosen domain name for Cozy: 10 cozy/fqdn doesn't exist
Installing NPM dependencies...
Creating UNIX users...
Generating CouchDB tokens...
Configuring Cozy Controller...
 is running
Waiting for Cozy Controller to start... (1/20)
Waiting for Cozy Controller to start... (2/20)
Waiting for Cozy Controller to start... (3/20)
Waiting for Cozy Controller to start... (4/20)
Waiting for Cozy Controller to start... (5/20)
Waiting for Cozy Controller to start... (6/20)
Waiting for Cozy Controller to start... (7/20)
Waiting for Cozy Controller to start... (8/20)
Waiting for Cozy Controller to start... (9/20)
Waiting for Cozy Controller to start... (10/20)
Waiting for Cozy Controller to start... (11/20)
Waiting for Cozy Controller to start... (12/20)
Waiting for Cozy Controller to start... (13/20)
Waiting for Cozy Controller to start... (14/20)
Waiting for Cozy Controller to start... (15/20)
Waiting for Cozy Controller to start... (16/20)
Waiting for Cozy Controller to start... (17/20)
Waiting for Cozy Controller to start... (18/20)
Waiting for Cozy Controller to start... (19/20)
Waiting for Cozy Controller to start... (20/20)
Cozy Controller is too long to start
dpkg: error processing package cozy (--configure):
 subprocess installed post-installation script returned error exit status 1
Setting up libldap-2.4-2:amd64 (2.4.31-1+nmu2ubuntu8.3) ...
Processing triggers for libc-bin (2.19-0ubuntu6.9) ...
Errors were encountered while processing:
 cozy
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

**Comment by QuentinPerez** [comment]
fixed with #21
